content/docs/user-guide/components/reference/vegetation/vegetation-instance-system.md

```diff
@@ -0,0 +1,53 @@
+---
+linkTitle: Vegetation Instance System
+title: Vegetation Instance System Component
+description: Manages and processes requests to create and destroy vegetation instances in Open 3D Engine (O3DE). 
+---
+
+The Vegetation Instance System component manages and processes requests to create and destroy vegetation instances, or static objects, throughout your world.
+
+Learn about how the Vegetation Instance System component works within the vegetation system in [Technical details](#technical-details) below.
+
+{{< note >}} 
+This is a system component that exists when you interact with the vegetation system via the Vegetation Gem. Generally, this system is already set up and you don't need to do anything further to manage vegetation in your world.
+{{< /note >}}
+
+## Provider
+
+[Vegetation Gem](/docs/user-guide/gems/reference/environment/vegetation/)
+
+## Source code
+
+[`Gems/Vegetation/Code/Source/InstanceSystemComponent.h`](https://github.com/o3de/o3de/blob/development/Gems/Vegetation/Code/Source/InstanceSystemComponent.h)
+
+
+## Properties
+
+| Property| Description | Values | Default |
+| --- | --- | --- | --- |
+| **Max Instance Process Time Microseconds** | The number of microseconds per tick for processing instance management tasks. | 0 to 33000 | 500 |
+| **Max Instance Task Batch Size** | The number of instance management tasks that can be batch processed together. | 1 to 2000 | 100 |
```

Comment:
This might be easier to understand if we clarify "instance management tasks". Maybe:
| **Max Instance Process Time Microseconds** | The number of microseconds per tick for processing instance spawning / despawning tasks. | 0 - 33000 | 500 |
	| **Max Instance Task Batch Size** | The number of individual instance spawn / despawn tasks that can be batch processed together. | 1 - 2000 | 100 |

Reply:
```suggestion
| **Max Instance Process Time Microseconds** | The number of microseconds per tick for processing instance spawning and despawning tasks. | 0 - 33000 | 500 |
| **Max Instance Task Batch Size** | The number of individual instance spawn and despawn tasks that can be batch processed together.| 1 - 2000 | 100 |
```